### ISSUE TITLE
[question/maybe bug] ariaLabel property in CommandBarButton is setting the same aria-label property in both the div wrapper of the split button and the inner button element

### ISSUE THREAD

**Issue by tomasro27** [OfficeDev/office-ui-fabric-react#10214]
### Environment Information

- **Package version(s)**: 7.24.0
- **Browser and OS versions**: chrome 75.0.3770.100 64-bit

### Describe the issue:

I'm not as familiar with accessibility so I'm not sure if this is a bug or it is the actual expected behavior.
This repros with `DefaultButton` and `CommandBarButton` when the property `split={true}`. So on any "Split Button".
The `ariaLabel` property is setting the same `aria-label` property in both the div that wraps the split button and the inner button element itself. In addition, the inner button already has a text describing the button. 
Is it okay to have a duplicate aria-label?

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/tomasro27/pen/RwboWGW

use dev tools to inspect the button and see the aria-label

#### Actual behavior:

Setting `ariaLabel` property in a "split button" results in `<div aria-label="label"><button aria-label="label"/></div>`
With the same label in both the div wrapper and the button itself. 
Also, the button is already labelled by a text string, in which case, `aria-labelledby` should be used instead of `aria-label` for the button.

#### Expected behavior:

If the button inside split button already has a label, maybe the button doesn't need an `aria-label`, but instead use `aria-labelledby`, referring to the text in the div inside the button.

In case there is no text on the first button on the Split button, then should the `aria-label` for the div container of the Split button be different than the `aria-label` of the first button?
Or maybe the div wrapper of the split button doesn't need an aria-label at all?

### Documentation describing expected behavior
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

In this case, the button already has a label, although the text is in a div inside the button. So I'm not sure whether the aria-label is needed or not.

**Comment by kenotron** [comment]
This looks like it potentially an issue, but since the label are actually just duplicated, it's not causing immediate issues with screen readers. I'll put this as a feature request.

**Comment by kenotron** [comment]
I don't think these are the same anymore - the labels are both labelled the same thing, but we add a word "split" in there to differentiate.

![image](https://user-images.githubusercontent.com/34725/65286488-98168980-daf4-11e9-9b0f-7cfac42c9e06.png)

**Comment by tomasro27** [comment]
So the "split" word is added to whatever string is passed through the aria-label property? If that's the case, I think this issue can be resolved.

I can see how other people would want to explicitly have an aria-label for each of those, but that is not my case.

thanks :)

**Comment by khmakoto** [comment]
If you need anything else related to this issue feel free to specify it here. If not, feel free to close the issue or let the bot close it for you. Thanks!
